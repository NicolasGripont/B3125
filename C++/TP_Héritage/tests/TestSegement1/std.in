S seg1 1 1 7 5
S seg1 2 3 4 5
S seg2 2 3 2 3
S seg3 -2 -4 2 10
S seg4 -2 -4 2 10 11
S seg4 -2 
HIT seg1 4 3 
HIT seg1 1 1
HIT seg1 7 5
HIT seg2 2 3
HIT seg1 0 0
HIT seg2 1 1
HIT seg3 0 3
DELETE seg1
S seg1 1 2 3 4
EXIT
