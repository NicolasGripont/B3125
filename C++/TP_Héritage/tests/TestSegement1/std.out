OK
ERR
OK
OK
ERR
ERR
YES
YES
YES
YES
NO
NO
YES
OK
OK